config/software/datadog-process-agent.rb

```diff
@@ -0,0 +1,15 @@
+name "datadog-process-agent"
+always_build true
+
+build do
+  agent_branch = ENV['PROCESS_AGENT_BRANCH']
+  if agent_branch.nil? || agent_branch.empty?
+    agent_branch = "master"
+  end
+
+  binary = "process-agent-amd64-#{agent_branch}"
```

Comment:
If you set `default_version` (expectedly `default_version agent_branch` immediately after collecting `agent_branch`) you can then use the `version` function here, it'd be a little more readable.